code/scoring/score.py

```diff
@@ -33,8 +33,8 @@ def init():
     global model
 
     # load the model from file into a global object
-    model_path = Model.get_model_path(
-        model_name="sklearn_regression_model.pkl")
+    model_path = os.path.join(os.getenv('AZUREML_MODEL_DIR'),
```

Comment:
This change causes the build fail. See the checks.

Reply:
Appears to be just linting issues

Reply:
**os** is used, but not imported